File: db/csharp/SalesManagement.Domain/Models/CategoryType.cs
namespace SalesManagement.Domain.Models
{
    /// <summary>
    /// 分類種別マスタ
    /// </summary>
    public class CategoryType
    {
        public string CategoryTypeCode { get; set; } = string.Empty;
        public string CategoryTypeName { get; set; } = string.Empty;
        public DateTime CreatedAt { get; set; }
        public string CreatedBy { get; set; } = string.Empty;
        public DateTime UpdatedAt { get; set; }
        public string UpdatedBy { get; set; } = string.Empty;
    }
}

File: db/fsharp/SalesManagement.Domain/Models/CategoryType.fs
namespace SalesManagement.Domain.Models

open System

/// <summary>
/// 分類種別マスタ（業種、規模などの分類軸を定義）
/// </summary>
[<CLIMutable>]
type CategoryType = {
    CategoryTypeCode: string
    CategoryTypeName: string
    CreatedAt: DateTime
    CreatedBy: string
    UpdatedAt: DateTime
    UpdatedBy: string
}
